pjsip/src/pjsip/sip_uri.c

```diff
@@ -404,7 +408,13 @@ static pj_status_t pjsip_url_compare( pjsip_uri_context_e context,
      * This includes userinfo containing passwords or formatted as 
      * telephone-subscribers.
      */
-    if (pj_strcmp(&url1->user, &url2->user) != 0)
+    str[0].ptr = buf[0];
```

Comment:
Should it protect this with PJSIP_UNESCAPE_WHEN_PARSING_URI when comparing unescape user part?